spec/telegraf/agent_spec.rb

```diff
@@ -16,9 +16,41 @@
 
       it 'merges tags into event' do
         # Write app tag here too to ensure it is overwritten by global tags
-        agent.write('series', tags: {app: 'no', field: 'yes'}, values: {a: 1})
+        agent.write!('series', tags: {app: 'no', field: 'yes'}, values: {a: 1})
         expect(socket_read).to eq 'series,app=test,field=yes,tagged=yes a=1i'
       end
     end
   end
+
+  describe '@before_send' do
+    let(:kwargs) { {before_send: before_send} }
+    let(:before_send) do
+      lambda {|data|
+        data.delete_if {|line| line[:tags].key?(:drop) }
+        data
+      }
+    end
+
+    it 'drops all points' do
```

Comment:
```suggestion
    it 'drops all points when all points match' do
```

Is that what you mean?